x11/pango/Portfile

```diff
@@ -2,32 +2,25 @@
 
 PortSystem              1.0
 PortGroup               active_variants 1.1
-PortGroup               gobject_introspection 1.0
+PortGroup               meson 1.0
 
 # Please keep the pango and pango-devel ports as similar as possible.
```

Comment:
Don't forget to regard this comment. Usually when a new stable version was out I would update pango-devel to a new version first, verify that it built on the buildbot on all OS versions, fix any problems I noticed, wait awhile to see if any bug reports came in about it, and then later update pango.

Reply:
Updated `pango-devel` via the following commit. (Per the commit message, this was done without a PR, to start validating these changes ASAP.)

https://github.com/macports/macports-ports/commit/51aea14775709205251c6004641f5d3996ed22a0